### ISSUE TITLE
error with status 999 on first login

### ISSUE THREAD

**Issue by wpcfan** [spring-projects/spring-authorization-server#852]
**Describe the bug**
The first time login to auth server will fail if using custom database schema

**To Reproduce**
1. start uaa-auth-server, todos-service and web-client module respectively
2. open browser and navigate to http://localhost:8081 (which is the web-client) and it is successfuly redirected to http://localhost:9000 (which is the authorization server)
3. use `user/12345678` to log in
4. you will see the following error

```json
{"timestamp":"2022-08-12T08:44:47.290+00:00","status":999,"error":"None","message":"No message available"}
```

And if we try to navigate to http://localhost:8081 again in the same browser window, the web-client index page will be displayed, the error is gone.

**Expected behavior**
The first login should be success

**Sample**

https://github.com/wpcfan/spring-auth-server.git

**Comment by jgrandja** [comment]
@wpcfan Please review the [messages-sample](https://github.com/spring-projects/spring-authorization-server/tree/main/samples#messages-sample) and compare it to your sample to see the differences. The messages-sample works so there is something different with your configuration.

**Comment by sjohnr** [comment]
Also @wpcfan, when using your local machine for development, make sure you launch the client with http://127.0.0.1:8080 (or http://127.0.0.1:8081 if you have changed the port). The client unfortunately can't use `localhost`. This is to ensure that there are two distinct hosts available for cookies (127.0.0.1, localhost) to avoid collisions, and because redirect uri validation rejects localhost as the host for a client. See gh-243 for more info.
